custom/icds_reports/utils/aggregation_helpers/distributed/agg_dashboard_activity.py

```diff
@@ -95,7 +95,8 @@ def get_user_locations(self):
 
             state_id, district_id, block_id, user_level = None, None, None, None
 
-            if user['location_id'] and user['location_id'] in self.transformed_locations:
+            if user['location_id'] and\
+                user['location_id'] in self.transformed_locations and user.get('is_active'):
```

Comment:
Nitpick: this would be more readable if you extracted `user['location_id'] and user['location_id'] in self.transformed_locations` into a well-named temp variable, plus then the `if` would fit on one line.

Reply:
cool. make sense.